allennlp/common/util.py

```diff
@@ -256,7 +256,7 @@ def prepare_environment(params: Params):
 
 
 def get_spacy_model(
-    spacy_model_name: str, pos_tags: bool, parse: bool, ner: bool
+    spacy_model_name: str, parse: bool, ner: bool, pos_tags: bool = True
```

Comment:
I don't like the idea of changing to order of arguments if it's not necessary. Can we just do this?

```suggestion
    spacy_model_name: str, pos_tags: bool = True, parse: bool = False, ner: bool = False
```

Reply:
Make sense :-)